cmd/iop/install.go

```diff
@@ -46,28 +48,26 @@ func installManifests(args *rootArgs) {
 
 	manifests, err := genManifests(args)
 	if err != nil {
-		logAndPrintf(args, "%s", err)
-		os.Exit(1)
+		log.Fatalf("Could not generate manifest: %v", err)
 	}
 
 	out, err := manifest.ApplyAll(manifests, version.NewVersion("", 1, 2, 0, ""), args.dryRun, args.verbose)
 	if err != nil {
-		logAndPrintf(args, "%s", err)
-		os.Exit(1)
+		log.Fatalf("Failed to apply manifest with kubectl client: %v", err)
 	}
 
 	for cn := range manifests {
 
 		cs := fmt.Sprintf("CompositeOutput for component %s:", cn)
-		logAndPrintf(args, "\n%s\n%s", cs, strings.Repeat("=", len(cs)))
+		log.Infof("\n%s\n%s", cs, strings.Repeat("=", len(cs)))
 		if out.Err[cn] != nil {
-			logAndPrintf(args, "Errors: %s\n", out.Err[cn])
+			log.Errorf("Error object: %s\n", out.Err[cn])
```

Comment:
Do we actually need to log both the error object and error strings?

Reply:
Yes, at least when error happens, we need to log both from kubectl. Maybe we can remove the one of them at later stage, but it is out of scope in this pr .